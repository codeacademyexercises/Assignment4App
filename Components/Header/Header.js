import React, { Component } from 'react';
import { Text, View } from 'react-native';
import styles from './Header.style';

export default class Header extends Component {
  render() {
    return (
      <View style={styles.HeaderContainer}>
        <Text style={styles.Header}>Awesome Forms</Text>
      </View>
    );
  }
}
