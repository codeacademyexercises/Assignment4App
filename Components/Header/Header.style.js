import {
  StyleSheet,
} from 'react-native';

export default StyleSheet.create({
  HeaderContainer: {
    height: 100,
    backgroundColor: '#4154AF',
    flexDirection: 'row',
    justifyContent: 'space-around',
  },
  Header: {
    color: 'white',
    fontSize: 20,
  },
});
