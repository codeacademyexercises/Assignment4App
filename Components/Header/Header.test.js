import React from 'react';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Header from './Header';

configure({ adapter: new Adapter() });

describe('<Header /> ', () => {
  it('snapshot test', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
