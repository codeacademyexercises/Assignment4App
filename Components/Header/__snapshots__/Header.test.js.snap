// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header />  snapshot test 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#4154AF",
      "flexDirection": "row",
      "height": 100,
      "justifyContent": "space-around",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 20,
      }
    }
  >
    Awesome Forms
  </Text>
</View>
`;
